sample/build.gradle

```diff
@@ -22,7 +22,7 @@ android {
 
 dependencies {
     compile "com.android.support:appcompat-v7:$SUPPORT_LIBRARY_VERSION"
-
+    compile "com.android.support:design:$SUPPORT_LIBRARY_VERSION"
```

Comment:
Oh sorry, tried some something out. And just overlooked this line. So you can delete. Sorry again :'(